torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -855,7 +855,7 @@ void BuildProfilerSubmodule(py::module* m) {
         {
           NoGilSection nogil;
           for (int i = 0; i <= timeout_s / interval_s; i++) {
-            status = tensorflow::profiler::pywrap::Trace(
+            status = tsl::profiler::CaptureRemoteTrace(
```

Comment:
@cliveverghese can you point us to the right openxla API to use in place of this line please?